usagov/usagov-2021
# usagov-1279-postcss: Update package-lock.json

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1279

## Description

Update of postcss dependency for https://github.com/usagov/usagov-2021/security/dependabot/128

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- Not a drupal module but a node module upgrade of postcss from 7.0.16 to 8.4.31. 
- Requires a new build
- Test to make sure everything looks correct.

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
